bait-files/src/test/java/com/google/sps/LoginServletTest.java

```diff
@@ -0,0 +1,85 @@
+package com.google.sps;
```

Comment:
should the test files be in the bait-files/src/test/java/com/google/sps/servlets path?

Reply:
Changed it up